### ISSUE TITLE
Proper device attributes tutorial

### ISSUE THREAD

**Issue by Bluscream** [rospogrigio/localtuya#1179]
The device setup screen is very confusing. What should i select where? The only thing i can guess is the "color mode" being "color" and the ID probably is the long hex string

**Comment by donburch888** [comment]
Yes it is rather confusing, because it is not intended for new users ;-(  Almost as confusing as your message...

- What device are you trying to setup ?
- Which setup screen ?
- which tutorial ? 
- Have you followed any of the tutorials on how to get dp_id's from Tuya website ?

**Comment by donburch888** [comment]
That is more helpful, thanks ... but no answer jumps out at me from that screen image :-(

Hopefully the values for "code" in the Tuya websites “Get Device Specific…” (note the capital letters) menu option should match up with the LocalTuya field names (such as the "brightness" in your image above).  Seeing only dp_id numbers and current values in LocalTuya is not very helpful. 

From Tuya website you should be able to summarise the info for your device similar to:
```
code            dp_id   type       values
switch_led,       20,   Boolean,
work_mode,        21,   Enum,      "white”,“colour”,“scene”,“music”
bright_value_v2,  22,   Integer,   10-1000
countdown_1,      26,   Integer,   0-86400
```
which I find so much more useful. 

I used this [YouTube video](https://www.youtube.com/watch?v=Q1ZShFJDvE0) to get local key and dp_id info from Tuya website; then discussion at https://community.home-assistant.io/t/localtuya-using-tuya-iot-platform-to-obtain-not-only-local-keys-but-also-dp-data-point-ids/423236.  Also be aware that on Tuya's website in  “Device Control” menu there are options “Get device specific…” and “Get Device Specific…” (note the capital letters) !!!  use the one with capital letters to get the dp_ids.
